drivers/hmis_external_apis/public/schemas/form_definition.json

```diff
@@ -203,83 +821,273 @@
             "HOH_AND_ADULTS",
             "VETERAN_HOH"
           ]
+        }
+      }
+    },
+    "groupFormNode": {
+      "$comment": "Group form nodes",
+      "allOf": [
+        {
+          "$ref": "#/$defs/baseFormNode"
         },
-        "type": {
-          "enum": [
-            "GROUP",
-            "BOOLEAN",
-            "CHOICE",
-            "CURRENCY",
-            "DATE",
-            "DISPLAY",
-            "TIME_OF_DAY",
-            "FILE",
-            "IMAGE",
-            "INTEGER",
-            "OBJECT",
-            "OPEN_CHOICE",
-            "STRING",
-            "TEXT"
+        {
+          "$ref": "#/$defs/dataCollectedAbout"
+        },
+        {
+          "type": "object",
+          "required": [
+            "item"
+          ],
+          "properties": {
+            "component": {
+              "enum": [
+                "DISABILITY_TABLE",
+                "HORIZONTAL_GROUP",
+                "INFO_GROUP",
+                "INPUT_GROUP"
+              ]
+            },
+            "item": {
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "$ref": "#/$defs/formNode"
+              }
+            },
+            "text": {
+              "type": "string"
+            },
+            "readonly_text": {
+              "type": "string"
+            },
+            "prefill": {
+              "$comment": "TODO: unclear what this means in a GROUP, add docs",
```

Comment:
`prefill` is only valid on groups, actually only on top-level groups. It's the flag that makes the "fill" button appear, that lets you select a previous assessment to populate from. It's only used on HUD assessments.